lxd/storage/drivers/utils_cgo.go

```diff
@@ -173,7 +173,7 @@ static int get_unused_loop_dev(char *name_loop)
 		return -1;
 
 	// try to open loop file with O_DIRECT flag, try again without if it fails
-	ret = open(name_loop, O_RDWR | O_CLOEXEC | O_DIRECT)
+	ret = open(name_loop, O_RDWR | O_CLOEXEC | O_DIRECT);
```

Comment:
This is something you introduced or that existed before?
If this existed before move this fix to the beginning of the series. If you did introduce it in an earlier commit don't introduce it in the first place. ;)